config/karma.base.js

```diff
@@ -107,7 +107,7 @@ const config = {
 
   client: {
     mocha: {
-      timeout: 20000,
+      timeout: 3000,
```

Comment:
I believe this line exists for firestore as well, if we can revert this timeout that'd be great.